### PR TITLE
Checklist update

### PR DESCRIPTION
**Pull request by KubaKonieczny**

### CODE REVIEW

File: Checklist.py [hunk 1]
```diff
@@ -2,15 +2,36 @@
...
+
+        #ustawienie w oknie
+        self.c.grid(row=0, column=0,  columnspan=3, sticky= W+E+N+S)
+        self.showButton.grid(row=2, column=1)
+        self.quitButton.grid(row=3, column=1)
 
     def close_windows(self):
-        self.master.destroy()
\ No newline at end of file
+          self.master.destroy()
```
Comment (BartoszSambor) [inline Checklist.py:34]: Po co ten tabulator?

Comment (KubaKonieczny) [reply Checklist.py:34]: Dobre pytanie

**Review by KubaKonieczny — COMMENTED** [review]
Super

**Comment by LukaszSuchan** [comment]
Super robota

**Comment by Avenives** [comment]
10/10

**Pull request merged**